src/Tools/dotnet-monitor/Auth/HashAlgorithmChecker.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Linq;
+
+namespace Microsoft.Diagnostics.Tools.Monitor
+{
+    class HashAlgorithmChecker
+    {
+        private static readonly string[] DisallowedHashAlgorithms = new string[]
```

Comment:
RIPEMD160 and MAC3DES were dropped. Intentional?

Reply:
Yes, this was discussed as PR feedback from the PR to implement temp api keys. I apparently forgot to "git add" the change when I committed my PR changes so I added that change into this PR.

Reply:
Done